civihr_employee_portal/views/civihr_employee_portal.views.inc

```diff
@@ -1117,29 +1117,61 @@ function removeAdditionalCommas($view) {
   return $fieldsToClean;
 }
 
-function removeInactiveDepartments($results = array()) {
-  foreach ($results as $key => $value) {
-    $startDate = !empty($value->hrjc_role_civicrm_contact_start_date) ? $value->hrjc_role_civicrm_contact_start_date : NULL;
-    $endDate = !empty($value->hrjc_role_civicrm_contact_end_date) ? $value->hrjc_role_civicrm_contact_end_date : NULL;
+/**
+ * Sets the name to NULL for relationships starting in future or ending in past
```

Comment:
The description says you're setting the name to NULL, but the function is actually removing the results with inactive managers. Is the description correct?